app/javascript/managers.js

```diff
@@ -11,21 +11,25 @@ export function canInstallUserJS() {
 }
 
 export async function canInstallUserCSS() {
-  if (localStorage.getItem('stylusDetected') == '1') {
+  if (localStorage.getItem('stylusDetected') === '1') {
```

Comment:
Don't make unrelated changes like this, just it makes it harder for me to figure out what's going on and makes the git history messy.